apps/src/applab/dataBrowser/dataStyles.js

```diff
@@ -15,12 +16,21 @@ export const row = {
 export const cell = {
   padding: cellPadding,
   border: '1px solid gray',
-  fontSize: 14
+  fontSize: 14,
+  maxWidth: 350,
+  overflow: 'hidden',
 };
 
-export const buttonCell = [cell, {
-  width: buttonCellWidth,
+export const addButtonCell = [cell, {
+  minWidth: addButtonCellWidth,
   backgroundColor: color.lightest_teal,
+  width: addButtonCellWidth,
+}];
+
+export const editButtonCell = [cell, {
+  minWidth: editButtonCellWidth,
+  backgroundColor: color.lightest_teal,
+  width: editButtonCellWidth,
```

Comment:
fixed in https://github.com/code-dot-org/code-dot-org/pull/10417/commits/1da506ad3ac1b0ac7b0c2dbd976c24023fb32af8